server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -148,6 +166,10 @@ protected void mirrorRemoteToLocal(File workDir, CommandExecutor executor,
                     final FileMode fileMode = treeWalk.getFileMode();
                     final String path = '/' + treeWalk.getPathString();
 
+                    if (ignoreNode.isIgnored(path, fileMode == FileMode.TREE) == MatchResult.IGNORED) {
```

Comment:
Food for thought: Should we strip the `remotePath` from the `path` to check if it's ignored or not?

Reply:
Would it be better to check only when `gitIgnore` is specified?
```java
@Nullable
private final IgnoreNode ignoreNode;

...
if (gitIgnore != null) {
    ignoreNode = new IgnoreNode();
    ignoreNode.parse(...);
}

...
if (ignoreNode != null && ...) {
    ...
}
```

Reply:
> Should we strip the remotePath from the path to check if it's ignored or not?

I think it should be.
And gitignore should be written in relative to `remotePath`
ex. remotePath is `/aaa` and excluded file is `/aaa/ex.txt`, gitignore will be `/ex.txt`
Let me try this way